sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -516,6 +518,15 @@ def __missing__(self, key):
 
 class _ParDoEvaluator(_TransformEvaluator):
   """TransformEvaluator for ParDo transform."""
+
+  def __init__(self, evaluation_context, applied_ptransform,
+               input_committed_bundle, side_inputs, scoped_metrics_container,
+               perform_dofn_pickle_test=True):
+    super(_ParDoEvaluator, self).__init__(
+        evaluation_context, applied_ptransform, input_committed_bundle,
+        side_inputs, scoped_metrics_container)
+    self._perform_dofn_pickle_test = perform_dofn_pickle_test
```

Comment:
<!--new_thread; commit:3d6511ca1799a602673dbf658e25d6877931a944; resolved:0-->
Please add comment that this workaround is for SDF.